contracts/credit/CreditConfiguratorV3.sol

```diff
@@ -62,10 +63,9 @@ contract CreditConfiguratorV3 is ICreditConfiguratorV3, ControlledTrait, Reentra
     }
 
     /// @notice Constructor
-    /// @param _acl ACL contract address
     /// @param _creditManager Credit manager to connect to
     /// @dev Copies allowed adaprters from the currently connected configurator
-    constructor(address _acl, address _creditManager) ControlledTrait(_acl) {
+    constructor(address _creditManager) ControlledTrait(IPoolV3(CreditManagerV3(_creditManager).pool()).acl()) {
```

Comment:
it's better to add sanity check for _creditManager is not zero address. We implemeted it in prev versions as:
_creditManager == address(0) ? address(0) : IPoolV3....